### PR TITLE
10.3.0 consolidated dropkick noattribs

### PR DESCRIPTION
**Pull request by Expressionless**
Build passing
This bundles definition entity attributes in to definitions when looking for SER_ entity attributes. Also adds a small QOL fix to SearchEntity when finding traits

### CODE REVIEW

**Review by varunshastry — APPROVED** [review]

File: kogitoq/gadaq/src/main/java/life/genny/gadaq/live/data/InternalConsumer.java [hunk 1]
```diff
@@ -1,10 +1,5 @@
 package life.genny.gadaq.live.data;
 
-import java.io.File;
-import java.io.IOException;
-import java.lang.invoke.MethodHandles;
-import java.nio.file.Files;
-import java.nio.file.StandardCopyOption;
```
Comment (Expressionless) [inline kogitoq/gadaq/src/main/java/life/genny/gadaq/live/data/InternalConsumer.java:7]: unused imports

File: qwandaq/src/main/java/life/genny/qwandaq/datatype/DataType.java [hunk 2]
```diff
@@ -38,8 +36,6 @@
 
 import life.genny.qwandaq.CoreEntityPersistable;
 import life.genny.qwandaq.serialization.CoreEntitySerializable;
-import life.genny.qwandaq.utils.CommonUtils;
-import org.apache.commons.lang3.StringUtils;
```
Comment (Expressionless) [inline qwandaq/src/main/java/life/genny/qwandaq/datatype/DataType.java:42]: unused imports

File: qwandaq/src/main/java/life/genny/qwandaq/entity/search/SearchEntity.java [hunk 3]
```diff
@@ -653,7 +653,10 @@ public <T extends Trait> Optional<T> getTrait(Class<T> trait, String traitCode)
 	@JsonbTransient
 	public <T extends Trait> List<T> getTraits(Class<T> traitType) {
 		List<T> t = traits.get(traitType);
-		if(t == null) return new ArrayList<>();
+		if(t == null){
+			t = new ArrayList<>();
+			traits.put(traitType, t);
+		}
```
Comment (Expressionless) [inline qwandaq/src/main/java/life/genny/qwandaq/entity/search/SearchEntity.java:659]: no point just returning a new arraylist if its not going to get linked up to something

File: dropkick/src/main/java/life/genny/dropkick/live/data/InternalConsumer.java [hunk 4]
```diff
@@ -147,7 +147,7 @@ public void getEvent(String event) {
 			target = qwandaUtils.generateProcessEntity(processData);
 			List<String> defCodes = processData.getDefCodes();
 			for (String defCode : defCodes) {
-				BaseEntity def = beUtils.getBaseEntity(defCode);
+				BaseEntity def = beUtils.getBaseEntity(defCode, true);
```
Comment (Expressionless) [inline dropkick/src/main/java/life/genny/dropkick/live/data/InternalConsumer.java:150]: get definition eas to iterate through

**Review by jasperRob — APPROVED** [review]

**Pull request merged**